### ISSUE TITLE
Heap Buffer Overflow #2

### ISSUE THREAD

**Issue by andrew-epstein** [pcrain/slippc#3]
Showing additional output this time:
```
$ ./src/slippc -i crashes/crash3.slp -d
Loading crashes/crash3.slp
Parsing header
  Slippi Header Matched
  Raw portion = 25730 bytes
Parsing event descriptions
  Event description length = 13 bytes
  Payload size for event 0x36: 417 bytes
  Payload size for event 0x37: 63 bytes
  Payload size for event 0x38: 37 bytes
  Payload size for event 0x39: 9 bytes
Parsing events proper
  Estimated 124 (+123) frames
  Warning: unknown event code 0x74 encountered
  Warning: unknown event code 0x61 encountered
  Warning: unknown event code 0x7b encountered
  Warning: unknown event code 0x55 encountered
  Warning: unknown event code 0x7 encountered
  Warning: unknown event code 0x73 encountered
  Warning: unknown event code 0x74 encountered
  Warning: unknown event code 0x61 encountered
  Warning: unknown event code 0x72 encountered
  Warning: unknown event code 0x74 encountered
  Warning: unknown event code 0x41 encountered
  Warning: unknown event code 0x74 encountered
  Warning: unknown event code 0x53 encountered
  Warning: unknown event code 0x55 encountered
  Warning: unknown event code 0x13 encountered
  Warning: unknown event code 0x32 encountered
  Warning: unknown event code 0x30 encountered
  Warning: unknown event code 0x31 encountered
  Warning: unknown event code 0x3a encountered
  Warning: unknown event code 0x30 encountered
  Warning: unknown event code 0x31 encountered
  Warning: unknown event code 0x3a encountered
  Warning: unknown event code 0x34 encountered
=================================================================
==95584==ERROR: AddressSanitizer: heap-buffer-overflow on address 0x61a000001690 at pc 0x00010a9c7dfe bp 0x7ffee5252ce0 sp 0x7ffee5252cd8
READ of size 8 at 0x61a000001690 thread T0
    #0 0x10a9c7dfd in slip::Parser::_parsePreFrame() parser.cpp:195
    #1 0x10a9b7dea in slip::Parser::_parseEvents() parser.cpp:118
    #2 0x10a9b378f in slip::Parser::_parse() parser.cpp:48
    #3 0x10a9b2ded in slip::Parser::load(char const*) parser.cpp:36
    #4 0x10a9ae986 in main main.cpp:28
    #5 0x7fff5e0793d4 in start (libdyld.dylib:x86_64+0x163d4)

Address 0x61a000001690 is a wild pointer.
SUMMARY: AddressSanitizer: heap-buffer-overflow parser.cpp:195 in slip::Parser::_parsePreFrame()
```

See details in #1.

**Comment by andrew-epstein** [comment]
The stack trace is pretty similar to #1 so the fix might be the same.

**Comment by pcrain** [comment]
Fixed; see comment in #1
